src/app/pages/common/entity/entity-wizard/entity-wizard.component.ts

```diff
@@ -259,7 +259,7 @@ export class EntityWizardComponent implements OnInit {
     }
   }
 
-  getSummaryValue(fieldConfig, formControl) {
+  getSummaryValue(fieldConfig: FieldConfig, formControl: any) {
```

Comment:
formControl should be of type `FormControl`

Reply:
Updated